helpers/deploymentConfig.ts

```diff
@@ -2628,6 +2678,18 @@ export const globalConfig: NetworkConfig = {
             supplySpeeds: ["771604938271604"], // 500 CRV over 90 days (648000 blocks)
             borrowSpeeds: ["1157407407407407"], // 750 CRV over 90 days (648000 blocks)
           },
+          {
+            asset: "XVS",
+            markets: ["FRAX"],
+            supplySpeeds: ["1481481481481481"], // 960 XVS over 90 days (648000 blocks)
+            borrowSpeeds: ["2222222222222222"], // 1440 CRV over 90 days (648000 blocks)
+          },
+          {
+            asset: "XVS",
+            markets: ["sFRAX"],
+            supplySpeeds: ["2222222222222222"], // 1440 CRV over 90 days (648000 blocks)
+            borrowSpeeds: ["1481481481481481"], // 960 XVS over 90 days (648000 blocks)
+          },
```

Comment:
This is going to deploy two new RewardsDistributor contract, but I think we could reuse the already deployed RewardsDistributor with XVS as a reward token and the Core pool as comptroller (`RewardsDistributor_Core_0`)

To do that, I would:
* move this config to the object at line 2669
* ensure no new RewardsDistributor contracts are deployed (remove the deployment files for RewardsDistributor 2 and 3)
* in the VIP, only add the command `setRewardTokenSpeeds`

do you think it would work?